### ISSUE TITLE
[Bug]: Unable to login user immediately  after registration

### ISSUE THREAD

**Issue by yesmeck** [FusionAuth/fusionauth-issues#2665]
### What happened?

Logging users immediately after registration results in the following error:

```json
{
  "applicationId": [
    {
      "code": "[invalid]applicationId",
      "message": "The [applicationId] property is not valid. No application exists with Id [076d3078-785e-41b7-8566-1accafa55f11]."
    }
  ]
}
```

Here is a reproduction repo https://github.com/yesmeck/fusionauth-login-bug

### Version

1.48.3

### Affects Versions

- 1.48.3
- 1.48.2
- 1.48.1

**Comment by yesmeck** [comment]
version 1.47.1 works fine.

**Comment by robotdan** [comment]
Can you provide a very specific set of recreate steps please?

**Comment by robotdan** [comment]
Internal:
- Try and recreate, possibly already fixed in `1.49.1`

**Comment by yesmeck** [comment]
I just tested 1.49.1, and the issue has been resolved.

**Comment by andrewpai** [comment]
Closing this issue as resolved in `1.49.1`.
